package.json

```diff
@@ -47,20 +47,15 @@
     "prepublish-package": "node ./scripts/rmLibDir",
     "publish-package": "npm run prepublish-package && npm run build && npm publish",
     "compodoc": "compodoc -w -p tsconfig.json -s -n \"huddly-sdk\" -d \"docs\" --minimal --customFavicon \"docs_style/assets/imgs/favicon.ico\" --disableGraph --disableInternal --hideGenerator --includesName \"Additional Information\"  --includes examples -a docs_style/assets -y docs_style",
-    "prepare": "npm run build",
+    "prepare": "npm run build && husky install",
```

Comment:
This will not work if `husky` is a dev dependency.

Reply:
What does `husky install` do?

Reply:
I'll look into it, I think I can just disable this for CI. Is there an env variable or something that makes it easy to tell if I'm currently in a CI environment?